src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -46,18 +46,28 @@ <h5>{{ version }}</h5>
              i18n-aria-label
              id="clusterStatus">
       <div class="d-flex flex-column justify-content-center align-items-center">
-        <ng-template #healthChecks>
-          <cd-health-checks [healthData]="healthData.health.checks"></cd-health-checks>
-        </ng-template>
+
         <ng-template #healthWarningAndError>
-        <div class="info-card-content-clickable mt-1"
+        <!-- <div > -->
+          <cds-toggletip
              [ngStyle]="healthData.health.status | healthColor"
-             [ngbPopover]="healthChecks"
+             [isOpen]="isOpen"
```

Comment:
we don't need to define a variable for it if we are not controlling what the variable does. So let's just directly put the false here and we can remove the typescript counterpart change.
```suggestion
             [isOpen]="false"
```